### PR TITLE
Minor tweaks to entity/results macros

### PR DESCRIPTION
**Pull request by tyom**
Generic tweaks for use across multiple entities.

### CODE REVIEW

File: src/apps/investment-projects/transformers.js [hunk 1]
```diff
@@ -34,20 +34,20 @@ function transformInvestmentProjectToListItem ({
...
-        isBadge: false,
       }, {
         name: 'sector',
         label: 'Sector',
         value: sector,
-        isBadge: false,
       },
-    ],
+    ].map(item => {
+      item.isBadge = !!item.isBadge
```
Comment (feedmypixel) [inline src/apps/investment-projects/transformers.js]: I've become a fan of using `Boolean(item.isBadge)` - thoughts?

File: src/templates/_macros/entities.njk [hunk 2]
```diff
@@ -35,7 +35,7 @@
       {% if metaItems | length %}
         <div class="c-entity__content">
           {% for metaItem in metaItems %}
-            {{ MetaItem(metaItem| assign({ highlightTerm: props.highlightTerm })) }}
+            {{ MetaItem(metaItem | assign({ highlightTerm: props.highlightTerm, isLabelHidden: metaItem.isLabelHidden })) }}
```
Comment (feedmypixel) [inline src/templates/_macros/entities.njk]: easier to read on a newline?

File: src/apps/investment-projects/transformers.js (hunk 1 above)
Comment (tyom) [inline src/apps/investment-projects/transformers.js]: Apparently there's a significant difference in performance in some browsers. While Chrome appears to perform slightly better with `Boolean`, Safari, for example, is much worse.

Comment (feedmypixel) [inline src/apps/investment-projects/transformers.js]: I'd air more on the clarity side of things, but not hung up about it 👍

**Pull request merged**